tests/operators/test_convection.py

```diff
@@ -3,22 +3,32 @@
 """Tests for convection diagnostics."""
 
 import iris
+import numpy as np
 
 import CSET.operators.convection as convection
 
 
 def test_cape_ratio():
     """Compare with precalculated ratio."""
-    precalculated = iris.load_cube("tests/test_data/convection/ECFlagB.nc")
-    precalculated_2 = iris.load_cube("tests/test_data/convection/ECFlagB_2.nc")
     SBCAPE = iris.load_cube("tests/test_data/convection/SBCAPE.nc")
     MUCAPE = iris.load_cube("tests/test_data/convection/MUCAPE.nc")
     MUCIN = iris.load_cube("tests/test_data/convection/MUCIN.nc")
-    assert (
-        convection.cape_ratio(SBCAPE, MUCAPE, MUCIN).data.all()
-        == precalculated.data.all()
-    )
-    assert (
-        convection.cape_ratio(SBCAPE, MUCAPE, MUCIN, MUCIN_thresh=-1.5).data.all()
-        == precalculated_2.data.all()
+    cape_75 = convection.cape_ratio(SBCAPE, MUCAPE, MUCIN)
+    precalculated_75 = iris.load_cube("tests/test_data/convection/ECFlagB.nc")
+    assert np.allclose(cape_75.data, precalculated_75.data, atol=1e-5, equal_nan=True)
+    # TODO: Test data clobbered by -75, so disabled until regenerated.
```

Comment:
Was this ever regenerated?

Reply:
I'm pretty sure we did. Wouldn't the test fail if we didn't?

I'll see if I can find it, and if not I'll produce it again.

Reply:
Thanks. If it all looks good then we can remove the comment.

Reply:
I've just updated my branch on my system based on latest commit, and the data of the two different files are there, so looks good, I'll remove the comment and then resolve this one.